src/riot-rs-embassy/src/thread_executor.rs

```diff
@@ -0,0 +1,180 @@
+use core::marker::PhantomData;
+use embassy_executor::{raw, Spawner};
+use riot_rs_threads::{current_pid, thread_flags, thread_flags::ThreadFlags, ThreadId};
+
+const THREAD_FLAG_EXECUTOR_WAKEUP: ThreadFlags = 0x01;
```

Comment:
Also, I just noticed the same value is used in `riot-rs-embassy::blocker`: https://github.com/future-proof-iot/RIOT-rs/blob/aa5296c8e2974b23519fbca951c865e34e7af6ba/src/riot-rs-embassy/src/blocker.rs#L6-L12  
Maybe not a problem since it is used in both cases to wakeup the thread, but still confusing; maybe unify it somewhere?

Reply:
Agreed, this needs unifying docimenting and reserving.

Reply:
Fixed in #313.